### PR TITLE
Switch to using new let-binding syntax.

### PR DESCRIPTION
**Pull request by nc6**
Addresses #259 for the chain spec. Also makes a couple of tidy-up changes along
the way.

### CODE REVIEW

File: specs/chain/latex/blockchain-spec.tex [hunk 1]
```diff
@@ -725,8 +726,7 @@ \section{Blockchain Extension}
   %
   \begin{equation*}
   \begin{array}{r@{~\in~}lr}
-    \var{data} & \Data    & \text{data}\\
-    \var{ds} & \DelegState & \text{ledger layer delegation state}
```
Comment (nc6) [inline specs/chain/latex/blockchain-spec.tex:729]: This is from the ledger spec. Other imported definitions are left undefined, so removing this for consistency

**Review by dnadales — APPROVED** [review]
So long overloaded `=` operator :cry:

**Pull request merged**